### PR TITLE
CORTX-28220 : Motr changes to support config format for partition stob config

### PR DESCRIPTION
**Pull request by mukundkanekar**
# Problem Statement
- Changes to support parse and update of partition stob configuration
    received via confd
- Note: There are hardcoded config string added for now, will be removed once hare is ready and provides required config.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x ] Basic testing with Motr+Hare cluster is done

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x ] JIRA number/GitHub Issue added to PR
- [x ] PR is self reviewed
- [x ] Jira and state/status is updated and JIRA is updated with PR link
- [x ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: motr/setup.c [hunk 1]
```diff
@@ -1473,342 +1473,26 @@ static int cs_be_dom_cfg_zone_pcnt_fill(struct m0_reqh          *reqh,
...
 
-		/**
-		 * Case 5: Multiple Data and no Meta specified
-		 */
-		return;
-	}
+        /* if (cs_conf_part_config_get(rctx, part_cfg, false) != 0)
+		return ; */
 
+        if (cs_conf_part_config_get(rctx, part_cfg, true) != 0)
```
Comment (vidyadhar-pinglikar) [inline motr/setup.c:1494]: return no useful here

File: ut/diter.cg [hunk 2]
```diff
@@ -13,7 +13,7 @@
     services=[service-0, service-1, service-2, service-3, service-4, service-5,
               service-6, service-7])
 (service-0 type=@M0_CST_IOS endpoints=["0@lo:12345:34:1"] params=[]
-    sdevs=[sdev-10, sdev-11, sdev-12, sdev-13, sdev-14])
+    sdevs=[sdev-10, sdev-11, sdev-12, sdev-13])
```
Comment (vidyadhar-pinglikar) [inline ut/diter.cg:16]: Changes in diter.cg need to be removed; are for optional testing; just revert this file

Comment (mukundkanekar) [reply ut/diter.cg]: reverted file.
commit be43731edd92782fb063b3f9e3d72d4cc078be29

File: motr/setup.c (hunk 1 above)
Comment (mukundkanekar) [reply motr/setup.c]: removed return and if condition.
commit 351218bfea6c47b27beb58f260b4485560c5872f

**Review by vidyadhar-pinglikar — APPROVED** [review]

**Pull request merged**